packages/backend/src/services/OrganizationService/OrganizationService.ts

```diff
@@ -325,6 +327,10 @@ export class OrganizationService {
             throw new ForbiddenError();
         }
 
+        if (data.emailDomains.some((domain) => isEmailProviderDomain(domain))) {
+            throw new ParameterError();
```

Comment:
We should also add this check in `packages/backend/src/services/UserService.ts` -> `completeUserSetup()`

Reply:
Sounds good! Although here I'm not sure what kind of error it should be. Maybe `ParametersError` with a description like `email domain is not valid to enable domain organization access`?

Reply:
Yes, that would work 👍 We can add a similar message in the organization service error.